### ISSUE TITLE
[Sugestion] A flag to deny players to sit their pets. 

### ISSUE THREAD

**Issue by matipoirierg** [aromaa/WorldGuardExtraFlags#242]
https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/entity/Sittable.html#setSitting(boolean)

**Comment by aromaa** [comment]
What would the use case be for this? Why do you need this?

**Comment by aromaa** [comment]
Right. This would be a problem if you have allowed interaction inside the region. This would fall under more general customizable interaction flags which are planned by EngineHub.

**Comment by matipoirierg** [comment]
> Right. This would be a problem if you have allowed interaction inside the region. This would fall under more general customizable interaction flags which are planned by EngineHub.

I see, didn't know interact would allow me to block this, ty very much.
